frontend/public/components/monitoring/alerting.tsx

```diff
@@ -585,180 +588,222 @@ const alertStateToProps = (state: RootState, { match }): AlertsDetailsPageProps
-                        </Link>
-                      </div>
-                    </dd>
-                  </dl>
-                </div>
-              </div>
-            </div>
+    const ele = (
+      <StatusBox data={alert} label={AlertResource.label} loaded={loaded} loadError={loadError}>
+        <h1 className="co-m-pane__heading">
+          <div data-test="resource-title" className="co-resource-item">
+            <MonitoringResourceIcon className="co-m-resource-icon--lg" resource={AlertResource} />
+            {labels?.alertname}
+            <SeverityBadge severity={labels?.severity} />
           </div>
-          {silencesLoaded && !_.isEmpty(alert?.silencedBy) && (
-            <div className="co-m-pane__body">
-              <div className="co-m-pane__body-group">
-                <SectionHeading text="Silenced By" />
-                <div className="row">
-                  <div className="col-xs-12">
-                    <Table
-                      aria-label="Silenced By"
-                      data={alert?.silencedBy}
-                      Header={silencesTableHeader}
-                      loaded={true}
-                      Row={SilenceTableRow}
-                    />
-                  </div>
-                </div>
-              </div>
+          {state !== AlertStates.Silenced && (
+            <div className="co-actions" data-test-id="details-actions">
+              <ActionButtons actionButtons={[silenceAlert(alert)]} />
             </div>
           )}
-        </StatusBox>
-      </>
+        </h1>
+        <HeaderAlertMessage alert={alert} rule={rule} />
+      </StatusBox>
+    );
+
+    const alertPodObj = {
+      namespace: alert?.labels.namespace,
+      kind: 'Pod',
+      kindObj: {
+        apiVersion: 'v1',
+        label: 'Pod',
+        labelKey: 'public~Pod',
+        plural: 'pods',
+        abbr: 'P',
+        namespaced: true,
+        kind: 'Pod',
+        id: 'pod',
+        labelPlural: 'Pods',
+        labelPluralKey: 'public~Pods',
+      },
+      name: alert?.labels.pod,
+      badge: null,
+    };
+
+    return (
+      <DetailsPage
+        alert={alert}
+        loaded={loaded}
+        rule={rule}
+        silencesLoaded={silencesLoaded}
+        loadError={props.loadError}
+        match={match}
+        namespace={alertPodObj.namespace}
+        kind={alertPodObj.kind}
+        kindObj={alertPodObj.kindObj}
+        name={alertPodObj.name}
+        //badge={alertPodObj.badge}
+        //getResourceStatus={podPhase}
+        ele={ele}
+        breadcrumbsFor={() => [
+          {
+            name: t('public~Alerts'),
+            path: namespace ? `/dev-monitoring/ns/${namespace}/alerts` : '/monitoring/alerts',
+          },
+          { name: t('public~Alert details'), path: undefined },
+        ]}
+        //{...props}
+        //getResourceStatus={podPhase}
+        //menuActions={menuActions}
+        pages={[navFactory.details(Details), navFactory.logs(AlertLogs)]}
```

Comment:
Instead of refactoring DetailsPage you can pass params to these pages like this 
```suggestion
        pages={[navFactory.details(() => <Details alert={alert} {...otherProps} />), navFactory.logs(AlertLogs)]}
```

Reply:
Thank you. I will make this change and this will also enable me not to change the HorizontalNav component.

Reply:
@divyanshiGupta I have made the necessary changes as suggested by you . Should i raise another PR for it?

Reply:
No don't need to raise another PR. I will try to review it today. 

Also the same alerts-page is used in devconsole monitoring so make sure that also works as expected. I will also try to run the code today. Let me know if additional setup steps are involved to see the logs.

Reply:
Thank you. No additional setup is required as of now to see the logs. Currently, All the logs corresponding to the pod and namespace affected by the alert are displayed there.  I will check if things work as expected on the devconsole monitoring.

Reply:
@divyanshiGupta Things are working as expected on devconsole monitoring as well. However, one issue i have experienced is that on the Alert Detail page for a particular alert, when switching from 'Details' tab to 'Logs' tab or vice-versa, then the corresponding component is not rendered. This issue is in both Administrator and Developer view on the console while switching the tabs. I was unable to resolve this.

Reply:
Okay. I will try to run your code today and see why that is happening and will let you know. After that issue is resolved we can raise a PR on upstream console.

Reply:
Thanks a lot @divyanshiGupta